public/src/components/channelManagement/epicTests/epicVariantPreview.tsx

```diff
@@ -99,21 +100,30 @@ const buildProps = (
   hasConsentForArticleCount: true,
 });
 
+const StorybookNames: Record<EpicModuleName, string> = {
+  ContributionsLiveblogEpic: 'components-marketing-contributionsliveblogepic--default',
+  ContributionsEpic: 'components-marketing-contributionsepic--default',
+};
+
 const useStyles = makeStyles(({}: Theme) => ({
   container: {
     width: '620px',
   },
+  iframe: {
+    width: '620px',
+    height: '800px',
```

Comment:
perhaps we can do something smarter than hardcoding dimensions

Reply:
👍 But fine to do in a future PR I think.